dist/core.bundle.js

```diff
@@ -56755,12 +56755,12 @@ return /******/ (function(modules) { // webpackBootstrap
 	
 	"use strict";
```

Comment:
Please DO NOT include the changes from dist/doc directories into a review anymore.